### PR TITLE
Add support for Rails 7 and Rack 3

### PR DESCRIPTION
**Pull request by n-rodriguez**
Thank you!

### CODE REVIEW

**Comment by n-rodriguez** [comment]
ping @adamcrown

**Comment by n-rodriguez** [comment]
ping @adamcrown

**Comment by n-rodriguez** [comment]
Hi there! Any news?

**Comment by adamcrown** [comment]
Apologies for being so very late on the reply here. I'm still trying to maintain some of these old gems I don't actively use. But I'm not doing such a great job of it recently.

This looks great though. Thank you!

**Comment by n-rodriguez** [comment]
Thank you!

File: voight_kampff.gemspec [hunk 1]
```diff
@@ -21,9 +21,9 @@ Gem::Specification.new do |s|
   s.test_files    = `git ls-files -- {tests}/**/*`.split("\n")
   s.require_path  = 'lib'
 
-  s.add_dependency 'rack', ['>= 1.4', '< 3.0']
+  s.add_dependency 'rack', ['>= 1.4']
```
Comment (AlexWayfer) [inline voight_kampff.gemspec:24]: JFYI, it's dangerous. It'd be better to leave the version lock, but upper. Like `< 4.0`.

File: voight_kampff.gemspec [hunk 2]
```diff
@@ -21,9 +21,9 @@ Gem::Specification.new do |s|
...
 
   s.add_development_dependency 'combustion', '~> 1.1'
-  s.add_development_dependency 'rails', '~> 5.2'
+  s.add_development_dependency 'rails', '>= 5.2'
```
Comment (AlexWayfer) [inline voight_kampff.gemspec:27]: This is also dangerous: you have no major version lock, as I remember. Any major version, 7, 8 or 9, can be breaking for this gem. So please add (bump the maximum) only when tested and working.

**Pull request merged**